wallet/README.md

```diff
@@ -185,6 +166,26 @@ total      : 90
 
 In this case we didn't specify a recipient of the new outputs, so the same default address was used. Next let's explore using some other keys.
 
+### Minting coins
+
+It is possible to mint new coins with the wallet.
+We can optionally pass the amount and public key of the owner as arguments to mint_coins.
+If optional arguments are not passed below are the default values:
+the amount is 100 coin
```

Comment:
The end of the previous section says, "Now let's explore using some other keys." And before this PR it went on to the section called "Using your own keys". But now you have inserted a section about minting in between so that transition does not make sense any more. I think minting should go first or last.